# Source: Written by Anders Ladegaard Marchsteiner.
# Upstream: No.
# Reason: Compile with newer ffmpeg.
diff -urpN a/src/plugins/thumbnailffmpeg_extractor.c b/src/plugins/thumbnailffmpeg_extractor.c
--- a/src/plugins/thumbnailffmpeg_extractor.c	2011-01-14 18:44:55.000000000 +0000
+++ b/src/plugins/thumbnailffmpeg_extractor.c	2012-05-28 20:26:58.991041239 +0000
@@ -480,7 +480,7 @@ RETRY_PROBE:
   for (i=0; i<format_ctx->nb_streams; i++)
     {
       codec_ctx = format_ctx->streams[i]->codec;
-      if (codec_ctx->codec_type != CODEC_TYPE_VIDEO)
+      if (codec_ctx->codec_type != AVMEDIA_TYPE_VIDEO)
         continue;
       codec = avcodec_find_decoder (codec_ctx->codec_id);
       if (codec == NULL)
